File: PYTHON/Program/Shirt.py
    # Saya Tattha Maharany Yasmin Akbar dengan NIM 2201805 mengerjakan soal Latihan 2 
    # dalam Praktikum mata kuliah Desain dan Pemrograman Berbasis Objek, untuk keberkahan-Nya
    # maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamin.
    
class Shirt():
    __color = ""
    __sleeve_type = ""
    
    def __init__(this, color = "", sleeve_type = ""):
        this.__color = color
        this.__sleeve_type = sleeve_type
    
    def get_color(this): return this.__color
    def set_color(this, color): this.__color = color
    
    def get_sleeve_type(this): return this.__sleeve_type
    def set_sleeve_type(this, sleeve_type): this.__sleeve_type = sleeve_type

File: PHP/Program/Shirt.php
<!-- /* 
    Saya Tattha Maharany Yasmin Akbar dengan NIM 2201805 mengerjakan soal Latihan 2 
    dalam Praktikum mata kuliah Desain dan Pemrograman Berbasis Objek, untuk keberkahan-Nya
    maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamin.
*/ -->

<?php

class Shirt {
    private $color = "";
    private $sleeve_type = "";

    // constructor
    public function __construct($color = "", $sleeve_type = "") {
        $this->color = $color;
        $this->sleeve_type = $sleeve_type;
    }

    // getter and setter methods
    public function get_color() {
        return $this->color;
    }

    public function set_color($color) {
        $this->color = $color;
    }

    public function get_sleeve_type() {
        return $this->sleeve_type;
    }

    public function set_sleeve_type($sleeve_type) {
        $this->sleeve_type = $sleeve_type;
    }

    // destructor (not necessary in PHP)
    // public function __destruct() {
    //     // cleanup code here
    // }
}

?>
